frontend/javascripts/test/sagas/volumetracing_saga_integration.spec.js

```diff
@@ -64,14 +84,223 @@ test.serial(
   },
 );
 
-test.serial("Brushing/Tracing with a new segment id should update the maxCellId", t => {
-  const newCellId = 13371339;
+test.serial("Brushing/Tracing with a new segment id should update the maxCellId", async t => {
+  t.context.mocks.Request.sendJSONReceiveArraybufferWithHeaders.returns(
+    Promise.resolve({ buffer: new Uint16Array(32 ** 3), headers: { "missing-buckets": "[]" } }),
+  );
+
+  const paintCenter = [0, 0, 0];
+  const brushSize = 10;
+
+  const newCellId = 2;
+
+  Store.dispatch(updateUserSettingAction("brushSize", brushSize));
+  Store.dispatch(setPositionAction([0, 0, 0]));
+  Store.dispatch(setToolAction("BRUSH"));
   Store.dispatch(setActiveCellAction(newCellId));
-  Store.dispatch(startEditingAction([12, 12, 12], OrthoViews.PLANE_XY));
+  Store.dispatch(startEditingAction(paintCenter, OrthoViews.PLANE_XY));
+  Store.dispatch(addToLayerAction(paintCenter));
   Store.dispatch(finishEditingAction());
 
-  // maxCellId should be updated after brushing/tracing
-  getVolumeTracingOrFail(Store.getState().tracing).map(tracing => {
-    t.is(tracing.maxCellId, newCellId);
-  });
+  t.is(await t.context.api.data.getDataValue("segmentation", paintCenter), newCellId);
```

Comment:
You've probably considered this but just in case you are not aware. The `getDataFor2DBoundingBox` api function could also be used here to get a full snapshot comparison of a particular bounding box. Maybe even using ava's snapshot functionality :thinking:

Reply:
Cool idea! Didn't think of that. I added it now.